NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift

```diff
@@ -10,10 +10,10 @@ import UIKit
 
 class NVActivityIndicatorAnimationAudioEqualizer: NVActivityIndicatorAnimationDelegate {
     
-    func setUpAnimationInLayer(layer: CALayer, size: CGSize, color: UIColor) {
+    func setUpAnimationInLayer(_ layer: CALayer, size: CGSize, color: UIColor) {
         let lineSize = size.width / 9
-        let x = (layer.bounds.size.width - lineSize * 7) / 2
-        let y = (layer.bounds.size.height - size.height) / 2
+        let x = (layer.bounds.width - lineSize*7)/2
+        let y = (layer.bounds.height - size.height)/2
```

Comment:
Please keep convention consistently. It should be whitespace between operand and operator.